core/datastorage/src/main/java/com/cognifide/aet/vs/mongodb/MongoDBClient.java

```diff
@@ -103,7 +102,17 @@ public void deactivate() {
 
     return companies;
   }
-  
+
+  protected void setupConfiguration(MongoDBClientConf config) {
+    if (StringUtils.isNoneBlank(config.mongoURI())) {
```

Comment:
You can use `isNotBlank` instead of `isNoneBlank` here :)

Reply:
Typo - thanks, fixed.